mu4e/mu4e-notification.el

```diff
@@ -73,9 +73,11 @@ zero."
   (when-let* ((fav (mu4e-bookmark-favorite))
               (delta-unread (plist-get fav :delta-unread)))
     (when (and (fboundp 'notifications-close-notification)
+               (fboundp 'dbus-ignore-errors)
                mu4e--notification-id
                (zerop delta-unread))
```

Comment:
when would `dbus-ignore-errors` _not_ be defined? Some emacs version, or in general when DBUS-support is not available?